White furry pie patch 4

WhiteFurryPie opened the pull request.

McCsocks commented:
Agree 2/2

mimosacb commented:
Agreed 3/3

stolksdorf commented:
Agreed 4/4. I like the language cleanup. Also, the two parter on that joke got me.

DukeKataron commented:
Agree 5/5

stolksdorf commented:
Rule has majority! Rule passes! Whew, just in time.

labria committed:
Add myself

WhiteFurryPie committed:
Adding Oleg

WhiteFurryPie committed:
Merge pull request #1 from labria/WhiteFurryPie-patch-1

Adding Oleg

WhiteFurryPie committed:
Update rules.md

Add democratic change in the newfound rules in order to make a better modified one.
